### DISCUSSION TITLE
Parenthesis matching colour

### DISCUSSION THREAD

**Discussion by JrgMyr** [pawelsalawa/sqlitestudio#4002] [Ideas]
Thanks for fixing the paranthesis matching function in the SQL editor pane.
One minor improvement would be to give it a different colour than that of marked text.
I would prefer to differentiate between "text marked" and "paranthesis matched".

**Comment by pawelsalawa** [comment]
Yeah, colors configuration was removed in 3.3.0 due to issues with dark/bright styles. I will think about reintroducing it, but I need to figure out a good way to make it work well with bright/dark styles.

**Reply by pawelsalawa** [reply]
#4006

**Reply by JrgMyr** [reply]
Color configuration is the big solution. Maybe something for a bigger version number later.
I would already be happy with "parenthesis matching" in a different color, say red or green or even bold or underline.
Just different from "selected text" in white on blue. That could be a quick change in 3.3.1. (I'm dreaming).

**Comment by JrgMyr** [comment]
Thanks.
Done.
